src/public.rs

```diff
@@ -7,44 +7,61 @@
 // except according to those terms.
 
 //! This module provides a `PublicKey` wrapping
-//! struct around a `ProjectivePoint` element.
+//! struct around a `AffinePoint` element.
```

Comment:
Small change: could be "around an `AffinePoint` element"

Reply:
Right, missed it! Thanks